ksp_plugin/interface_planetarium.cpp

```diff
@@ -96,7 +100,18 @@ Planetarium* __cdecl principia__PlanetariumCreate(
       world_to_plotting_affine_map * camera_to_world_affine_map,
       focal * Metre);
 
-  return m.Return(plugin->NewPlanetarium(parameters, perspective).release());
+  return m.Return(
+      plugin->NewPlanetarium(
+          parameters,
+          perspective,
+          [plotting_to_world = world_to_plotting_affine_map.Inverse(),
```

Comment:
Declare a variable for the lambda before the return statement so that we have a clue what it does and how to parse it.